src/engraving/rw/write/twrite.cpp

```diff
@@ -919,8 +919,13 @@ void TWrite::write(const Chord* item, XmlWriter& xml, WriteContext& ctx)
     } else if (item->stem() && (item->stem()->isUserModified() || !item->stem()->userLength().isZero())) {
         write(item->stem(), xml, ctx);
     }
-    if (item->hook() && item->hook()->isUserModified()) {
-        write(item->hook(), xml, ctx);
+    if (item->hook()) {
+        if (item->hook()->isUserModified()) {
+            write(item->hook(), xml, ctx);
+        }
+        if (item->hookIsReversed()) {
+            writeProperty(item, xml, Pid::HOOK_REVERSED);
+        }
```

Comment:
Maybe this needs to get added to read302.cpp too, in order to be able to read this from 3.7 scores